v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
```

Comment:
The arrow icon should rotate in position. Currently it looks like the image is being swapped and the baseline for the two images is different

https://user-images.githubusercontent.com/18233774/224711392-b1f8255c-8ed0-45e6-bdc0-67ffdca111bd.mov

Reply:
This is still pending